.github/workflows/cicd.yml

```diff
@@ -82,6 +102,12 @@ jobs:
         run: |
           npm publish
 
+      - name: Publish Release Notes
+        env:
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          npm run release:post
```

Comment:
would a  `postpublish` lifecycle script would work here? I like the explicit nature of this too, so no action is required

Reply:
I thought about your feedback a bit and decided to retain the current functionality as it'd determine the success or failure of the release by the success or failure of publishing the notes.

Current (pseudo):
```js
// Validate
await install()
await lint()
await build()

if (GITHUB_BRANCH === 'main') {
  await checkReleaseForSkip()

  await configureGit()
  await validateGitConnection()
  await createGitRelease()
  await pushGitRelease()
  await configureNpm()
  try {
    await publishNpmPackage()
  } catch {
    await revertLastTag()
    await revertLastCommit()
    return process.exit(1)
  }

  await publishReleaseNotes()
}
```

With `postpublish`:
```js
// Validate
await install()
await lint()
await build()

if (GITHUB_BRANCH === 'main') {
  await checkReleaseForSkip()

  await configureGit()
  await validateGitConnection()
  await createGitRelease()
  await pushGitRelease()
  await configureNpm()
  try {
    await publishNpmPackage()
      .then(() => publishReleaseNotes())
  } catch {
    await revertLastTag()
    await revertLastCommit()
    return process.exit(1)
  }
}
```

Under the latter scenario, successfully publishing but failing to create release notes would cause the rollback.